pkg/storage/util/util.go

```diff
@@ -132,3 +136,13 @@ func GenerateStorageName(listers *regopclient.StorageListers, additionalInfo ...
 
 	return strings.ToLower(name), nil
 }
+
+// Helper function to test featuregate ChunkSizeMiB
+func ChunkSizeMiBFeatureGateEnabled(featureGateAccessor featuregates.FeatureGateAccess) (bool, error) {
```

Comment:
Where is this function used? 🤔

Reply:
here --> https://github.com/openshift/cluster-image-registry-operator/pull/1026/commits/933c26e3290824dc55319dd270ff07ad3ab2c046#diff-fcbc7c17a683424f7d245a7af42669bc401010621d8fccfa1da7860e152e20d3R512